Formula/a/ansible-cmdb.rb

```diff
@@ -1,21 +1,54 @@
 class AnsibleCmdb < Formula
+  include Language::Python::Virtualenv
+
   desc "Generates static HTML overview page from Ansible facts"
   homepage "https://github.com/fboender/ansible-cmdb"
-  url "https://github.com/fboender/ansible-cmdb/releases/download/1.31/ansible-cmdb-1.31.tar.gz"
-  sha256 "ab1be4a85184647bcec08d4e65bda66c7d08c0f88c81eca4d0e44e02b768c2bb"
+  url "https://github.com/fboender/ansible-cmdb/archive/refs/tags/1.31.tar.gz"
+  sha256 "8de9a02e3f0740967537850f6263756dca1bf506cd95c1f2ef7f4ee6d9ff23b8"
   license "GPL-3.0-or-later"
 
   bottle do
     sha256 cellar: :any_skip_relocation, all: "39eadb125103aac400189e36112333e6e1da17fb75eae26ec1dd34d99185d798"
   end
 
   depends_on "libyaml"
+  depends_on "python@3.11" # cannot upgrade > 3.11 since release 1.31 uses the "imp" library
```

Comment:
any upstream issue for this?

Reply:
Yes, I'll add it in the comment: https://github.com/fboender/ansible-cmdb/issues/259

Reply:
since it's a single import, wdyt about adding this patch: https://github.com/fboender/ansible-cmdb/pull/260/commits/02242d4eed9d4295d02cf2835a51eb4f422b18cf (from https://github.com/fboender/ansible-cmdb/pull/260)

Reply:
That works. I can now use `python@3.13`. Thanks!